riff-raff/app/controllers/authentication.scala

```diff
@@ -87,14 +87,14 @@ class Login(deployments: Deployments, val controllerComponents: ControllerCompon
   }
 
   def profile = authAction { request =>
-    val records = deployments.getDeploys(Some(DeployFilter(deployer=Some(request.user.fullName)))).logAndSquashException(Nil).reverse
+    val records = deployments.getDeploys(Some(DeployFilter(deployer=Some(request.user.fullName)))).map(_.reverse)
     Ok(views.html.auth.profile(request, records))
   }
 
   val authorisationForm = Form( "email" -> nonEmptyText )
 
   def authList = authAction { request =>
-    Ok(views.html.auth.list(request, Persistence.store.getAuthorisationList.sortBy(_.email)))
+    Ok(views.html.auth.list(request, Persistence.store.getAuthorisationList.map(_.sortBy(_.email))))
```

Comment:
Rather than pushing this down into each controller, I'd consider using one of the `views.errorPage` or `views.errorContent` templates. It should also use a 5xx status code.